src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -146,7 +146,9 @@ public sealed class LegacyPackageReferenceProject : BuildIntegratedNuGetProject
                 LibraryRange = new LibraryRange(
                     name: packageId,
                     versionRange: range,
-                    typeConstraint: LibraryDependencyTarget.Package)
+                    typeConstraint: LibraryDependencyTarget.Package),
+                SuppressParent = __.SuppressParent,
```

Comment:
fix formatting

Reply:
this is the right formatting, these fields are outside of LibraryRange